composer.json

```diff
@@ -0,0 +1,7 @@
+{
+	"name": "woocommerce/wc-api-dev",
```

Comment:
The composer.json schema recommends avoiding non alphanumeric characters in the name https://getcomposer.org/doc/04-schema.md#name